### PR TITLE
Bookstore: Building Blocks

### PR DESCRIPTION
**Pull request by jasonfelice**
### Bookstore Milestone #1

- [x] Created the following pages:
  - Books
  - Categories

- [x] Created `Header`, `Book`, & `Add Books` components.
- [x] Give the webpage a basic styling.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <>
+      <div className={styles.book_wrapper}>
+        <div className={styles.content_wrapper}>
+          <div className={styles.content}>
+            <span className={styles.title}>{title}</span>
+            <span className={styles.author}>{author}</span>
+          </div>
+          <div className={styles.actions}>
+            <button type="button">Remove</button>
```
Comment (OLIPLICHE) [inline src/components/Book.js:17]: - [x] Please display the remove button. the button wouldn't remove the books yet, this is just a placeholder for now. Kindly follow the requirements.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @jasonfelice , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (jasonfelice) [reply src/components/Book.js:17]: Hello @OLIPLICHE, 
Hope you're having a delightful day.

I am not sure what you mean by displaying the remove button. For the remove button to be visible, I have to use the Book component on the `Books` page. Is that what you mean?

Thanks!

Comment (OLIPLICHE) [reply src/components/Book.js:17]: - [ ] Please, Just the way you displayed `Add New Book` from the component

**Review by ahmetbozaci — APPROVED** [review]
# STATUS: APPROVED 🥳 🥳 🥳 


@jasonfelice  ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by jasonfelice** [comment]
Thank You, @ahmetbozaci

**Pull request merged**